test/utils/ERC20PermitHarness.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.23;
+
+import { ERC712 } from "../../src/libs/ERC712.sol";
+
+import { ERC20Permit } from "../../src/ERC20Permit.sol";
+
+contract ERC20PermitHarness is ERC20Permit {
```

Comment:
Harnesses are not supposed to have logic. If they do, it means your tests are testing the harness functions, and not focusing on only what is inn the `src`.

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/7/commits/22b023f017e27adf3e859da12aac5e4266191c4b